version.props

```diff
@@ -1,11 +1,14 @@
 ﻿<Project>
   <PropertyGroup>
     <VersionPrefix>1.1.0</VersionPrefix>
-    <JavaClientVersion>0.1.0-preview1</JavaClientVersion>
+    <JavaVersionPrefix>0.1.0</JavaVersionPrefix>
     <VersionSuffix>preview1</VersionSuffix>
     <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' == 'rtm' ">$(VersionPrefix)</PackageVersion>
     <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' != 'rtm' ">$(VersionPrefix)-$(VersionSuffix)-final</PackageVersion>
     <BuildNumber Condition="'$(BuildNumber)' == ''">t000</BuildNumber>
+    <JavaPackageVersion>$(JavaVersionPrefix)-$(VersionSuffix)-$(BuildNumber)</JavaPackageVersion>
```

Comment:
So we'll get the `t000` suffix when there's no build number here, right? 
I remember running into issues with that,

Reply:
Oh, I remember now. I added the `preview1` to the `JavaClientVersion` because using the version suffix variable was appending the placeholder build number.
Disregard the previous comment